src/components/Tabs/Tabs.tsx

```diff
@@ -181,6 +185,48 @@ export const Tabs = ({
     );
   }, [tabs, getUID]);
 
+  /**
+   * Handles if scroll fade indicators should be displayed.
+   */
+  const handleTabsScroll = debounce(
+    (headerEl: HTMLDivElement) => {
+      const scrollLeft = headerEl.scrollLeft;
+      const width = headerEl.clientWidth;
+      const scrollWidth = headerEl.scrollWidth;
+
+      if (scrollLeft > 0) {
+        setScrollableLeft(true);
+      } else {
+        setScrollableLeft(false);
+      }
+
+      if (scrollWidth > width && scrollLeft + width < scrollWidth) {
+        setScrollableRight(true);
+      } else {
+        setScrollableRight(false);
+      }
+    },
+    100,
+    { leading: true },
+  );
+
+  /**
+   * Listens for window resize to display scroll fade indicators.
+   */
+  useEffect(() => {
+    if (headerRef && headerRef.current) {
+      const resizeHandleTabs = () => {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        handleTabsScroll(headerRef.current!);
+      };
+      resizeHandleTabs();
```

Comment:
The event listener actually calls the callback once when initiated, but the event listener is not triggered with prop changes so this line is required.
Sure it may be called twice on initial paint but that's better than it not being called at all.

Reply:
Maybe add something to that effect as a comment? Tracking down bugs associated with react lifecycle event triggers can be pretty annoying.

Reply:
Makes sense to me, will add